platform/commonUI/edit/src/actions/CancelAction.js

```diff
@@ -82,7 +66,7 @@ define(
         CancelAction.appliesTo = function (context) {
             var domainObject = (context || {}).domainObject;
             return domainObject !== undefined &&
-                domainObject.hasCapability("editor");
+                domainObject.getCapability("status").get("editing");
```

Comment:
Now that `editor` is generally applicable (which I think is a good change) I think it would be useful to use this instead of the `editing` status for this kind of check. `domainObject.hasCapability('editor') && domainObject.getCapability('editor').isEditingContextRoot()`, for instance. (Believe this would solve the quirkiness of checking status sometimes, capabilities other times, which we've noticed before.)